File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = s.split(' ') # 문자열을 공백을 기준으로 나눠 리스트로 만듦
    
    for i in range(len(answer)):
        if answer[i]:
            answer[i] = answer[i][0].upper() + answer[i][1:].lower()
        else:
            continue
        
    return " ".join(answer)

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    s = s.toLowerCase()
    const words = s.split(' ')
    const convert = words.map((word) => {
        const first = word.substring(0, 1).toUpperCase()
        const rest = word.slice(1).toLowerCase()
        return first + rest
    })
    return convert.join(" ");
}